packages/api/src/services/tags.service.ts

```diff
@@ -24,4 +28,36 @@ export class TagsService {
         await this.tagRepository.insert(tag);
         return this.tagRepository.findOneBy(tag);
     }
+
+    async update(updateTagNameRequest: UpdateTagNameRequest): Promise<TransactionTag> {
+        // This statement checks if the transactionTag that we are requesting to update
+        // exists in the database
+        if (
+            !(await this.tagRepository.findOneBy({
+                userId: updateTagNameRequest.userId,
+                tagName: updateTagNameRequest.currentName,
+            }))
+        ) {
+            throw new BadRequestException();
```

Comment:
If you add a descriptive error message here, you can remove that comment above since it becomes self explanatory

```suggestion
            throw new BadRequestException("Transaction tag not found");
```